sky/engine/bindings/snapshot_generic.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file serves as a generic snapshot entry point. It used to generate
+// the blobs necessary to create valid kDartIsolateSnapshotBuffer and
+// kDartVmIsolateSnapshotBuffer symbols for VM initialization. 
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:core';
+import 'dart:developer';
+import 'dart:io';
+import 'dart:isolate';
+import 'dart:math';
+import 'dart:mojo.internal';
+import 'dart:ui';
+import 'dart:typed_data';
```

Comment:
I could possibly set a custom URL mapping, but I would need to do that for a path that does not make sense on iOS.